question-servers/freeform.js

```diff
@@ -165,7 +165,12 @@ module.exports = {
             const resolvedElement = module.exports.resolveElement(elementName, context);
             const cwd = resolvedElement.directory;
             const controller = resolvedElement.controller;
-            const pythonArgs = [elementHtml, data];
+
+            let dataCopy = _.cloneDeep(data);
+            if ('base_url' in data.options) {
+                dataCopy.options.client_files_element_url = path.join(data.options.base_url, "elements", elementName, 'clientFilesElement');
```

Comment:
when running on Windows, this would give us backslashes in a URL: https://nodejs.org/api/path.html#path_windows_vs_posix. consider `path.posix.join` or just a template literal like
```js
`${data.options.base_url}/elements/${elementName}/clientFilesElement`
```